adlfs/spec.py

```diff
@@ -749,7 +749,10 @@ async def _info(self, path, refresh=False, **kwargs):
         kwargs["version_id"] = version_id
 
         out = await self._ls(
-            self._parent(fullpath), invalidate_cache=invalidate_cache, **kwargs
+            self._parent(fullpath),
+            detail=True,
```

Comment:
Context: we were returning infos by default before, but now `detail` is supported and is `False` by default, so we need to manually ask for detailed results here and in a few other places below.